config/webpack/webpack.common.js

```diff
@@ -30,6 +30,8 @@ module.exports = {
             patterns: [
                 {from: 'web/favicon.png'},
                 {from: 'web/favicon-unread.png'},
+                {from: 'web/icon-square.png'},
+                {from: 'web/icon-square@4x.png'},
```

Comment:
I think we only need one of these, right? Also, could you rename the image file to something more descriptive like `og-preview-image.png`